### PR TITLE
Only keep approved branches up to date

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
We're running a ton of unnecessary CI jobs and this doesn't provide us a
ton of value at the moment as we only merge a few PRs a day.

We can always loosen the requirement in the future in need-be.

### CODE REVIEW

**Comment by brettlangdon** [comment]
I don't think we need to be this strict

**Review by P403n1x87 — APPROVED** [review]
👍

**Review by Yun-Kim — DISMISSED** [review]

File: .mergify.yml [hunk 1]
```diff
@@ -8,14 +8,11 @@ pull_request_rules:
...
-  # approval. This makes it more likely to be ready to be merged once the
-  # second approval comes.
   - name: prepare for merge
     conditions:
       - label!=manual merge
       - check-success=test
-      - "#approved-reviews-by>=1"
+      - "#approved-reviews-by>=2"
     actions:
       update: {}
```
Comment (jd) [inline .mergify.yml:17]: You can remove this rule altogether then, the `strict: smart` does it automatically.

**Review by jd — CHANGES_REQUESTED** [review]

**Comment by Kyle-Verhoog** [comment]
Going to close this as it looks like we're ok with things as-is

**Pull request closed without merging**